bin/main.rb

```diff
@@ -1,2 +1,68 @@
 #!/usr/bin/env ruby
-puts 'Hello world!'
+
+puts 'WELCOME TO TIC-TAC-TOE GAME !!!'
+puts
+attr_accessor :player_one, :player_two, :board # Getter and Setter methods
+
```

Comment:
## Used only in a class
- [ ] Remove `attr_accessor :player_one, :player_two, :board # Getter and Setter methods` because is only use with classes